pkg/network/protocols/postgres/model_linux.go

```diff
@@ -73,6 +73,14 @@ func (e *EventWrapper) Operation() Operation {
 	return e.operation
 }
 
+// extractParameters returns the string following the command
+func (e *EventWrapper) extractParameters() string {
+	b := getFragment(&e.Tx)
+	idxStart := bytes.Index(b, []byte(" ")) + 1 // trim operation
+	idxEnd := bytes.Index(b, []byte("\x00"))    // trim trailing nulls
+	return string(b[idxStart:idxEnd])
```

Comment:
Isn't this going to panic on malformed input because the slice ranges will be invalid (-1, end smaller than start, etc)?

Reply:
I've changed this function in last commit to verify returned indices. Tnx.